compiler/rustc_const_eval/src/interpret/intrinsics.rs

```diff
@@ -513,7 +522,9 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
                 let op = self.eval_operand(op, None)?;
                 let cond = self.read_scalar(&op)?.to_bool()?;
                 if !cond {
-                    throw_ub_format!("`assume` called with `false`");
+                    throw_ub!(Custom(CustomSubdiagnostic::label(|| {
+                        fluent::const_eval_assume_false
+                    })));
```

Comment:
how is `throw_ub_custom!` different from this?

Reply:
They are the same. I wrote these before `throw_ub_custom!`.